commons/src/main/java/com/ge/predix/acs/commons/policy/condition/groovy/GroovyConditionShell.java

```diff
@@ -102,23 +101,23 @@ private static SecureASTCustomizer createSecureASTCustomizer() {
         // Disallow all imports by setting a blank whitelist.
         secureASTCustomizer.setImportsWhitelist(Arrays.asList(new String[] {}));
         // Disallow star imports by setting a blank whitelist.
-        secureASTCustomizer.setStarImportsWhitelist(Arrays.asList(new String[] { "org.crsh.command.*", "org.crsh.cli.*",
-                "org.crsh.groovy.*", "com.ge.predix.acs.commons.policy.condition.*" }));
+        secureASTCustomizer.setStarImportsWhitelist(Arrays.asList(
```

Comment:
I think we should be ok. I imported the code style that we use in eclipse and tried to match all the rules in the Java Code style preferences in Intellij to what we have in Eclipse. If someone checks out code and it reformats differently we can take another look at the rules.